reference-architecture/osp-cli/OSv3.yml.template

```diff
@@ -0,0 +1,36 @@
+deployment_type: openshift-enterprise
```

Comment:
It looks like this is used in group_vars, if that is the case, it should be named `OSEv3.yml` to match the byo groups.